Directory.Packages.props

```
@@ -16,7 +16,7 @@
     <!-- Test Platform, .NET Test SDK and Object Model  -->
     <MicrosoftNETTestSdkVersion>17.12.0</MicrosoftNETTestSdkVersion>
     <MicrosoftPlaywrightVersion>1.49.0</MicrosoftPlaywrightVersion>
-    <MicrosoftTestingExtensionsFakesVersion>17.12.0-beta.24429.1</MicrosoftTestingExtensionsFakesVersion>
+    <MicrosoftTestingExtensionsFakesVersion>17.12</MicrosoftTestingExtensionsFakesVersion>
```

You missed the last digit `.0`. I need to add some basic acceptance test here to avoid such mistake.

```suggestion
    <MicrosoftTestingExtensionsFakesVersion>17.12.0</MicrosoftTestingExtensionsFakesVersion>
```